sys/linux/filesystem.txt

```diff
@@ -11,6 +11,8 @@ umount2(path ptr[in, filename], flags flags[umount_flags])
 mount$bpf(src const[0], dst ptr[in, filename], type ptr[in, string["bpf"]], flags flags[mount_flags], opts ptr[in, fs_options[bpf_options]])
 mount$overlay(src const[0], dst ptr[in, filename], type ptr[in, string["overlay"]], flags flags[mount_flags], opts ptr[in, fs_options[overlay_options]])
 mount$binder(src const[0], dst ptr[in, filename], type ptr[in, string["binder"]], flags flags[mount_flags], opts ptr[in, fs_options[binder_options]])
+mount$bind(src ptr[in, filename], dst ptr[in, filename], type ptr[in, string["pipefs"]], flags flags[mount_flags_bind], data const[0])
```

Comment:
Why is it "pipefs"? If this value is unused, maybe we better provide NULL/empty string?

Reply:
syzkaller checks that the filesystem is listed in /proc/filesystems. I used pipefs as a hack because the kernel just ignores this argument for bind mounts. Otherwise it would require an exception to the syzkaller code. :)

Reply:
ah, ok, then a comment would be useful, because seeing "pipefs" is a bit strange

Reply:
OK, I'll paste the commit comment.